src/test/java/com/hotels/beeju/HiveServer2JUnitRuleTest.java

```diff
@@ -110,7 +110,7 @@ public void createDatabase() throws Exception {
     String databaseName = "Another_DB";
 
     server.createDatabase(databaseName);
-    HiveMetaStoreClient client = server.newClient();
```

Comment:
put this method back on the rule and have it delegate, otherwise we've broken the public API.